### PR TITLE
Refactor merchant dashboard

### PR DESCRIPTION
**Pull request by cjkelling**
Merchant dashboard will always show orders for that merchant, no matter the order status. The dashboard also displays the status for each order.

### CODE REVIEW

File: app/models/merchant.rb [hunk 1]
```diff
@@ -28,6 +28,6 @@ def distinct_cities
   end
 
   def specific_orders
-    item_orders.group(:order_id).select('item_orders.order_id, sum(quantity) as order_total_quantity, sum(quantity * item_orders.price) as order_total_cost').where(status: 0)
+    item_orders.group(:order_id).select('item_orders.order_id, sum(quantity) as order_total_quantity, sum(quantity * item_orders.price) as order_total_cost')
```
Comment (lrs8810) [inline app/models/merchant.rb:31]: Thank you! This looks great.

File: spec/features/merchants/items/edit_spec.rb [hunk 2]
```diff
@@ -1,96 +1,98 @@
...
+        expect(@tire.image).to eq('https://images-na.ssl-images-amazon.com/images/I/51HMpDXItgL._SX569_.jpg')
+        expect(@tire.inventory).to eq(25)
+        expect(@tire.active?).to eq(true)
+
+        # Let's ask about this!
+
+        # within "#item-#{@tire.id}" do
+        #   expect(page).to have_content('Chamois Buttr')
+        #   expect(page).to have_content('Deactivate Item')
+        # end
```
Comment (CoopTang) [inline spec/features/merchants/items/edit_spec.rb:68]: We can remove this now that we've gotten our answer.

File: app/views/merchant/dashboard/index.html.erb [hunk 3]
```diff
@@ -1,16 +1,27 @@
...
 
+<br>
+
+<h3>Orders:</h3>
+
 <% @merchant.specific_orders.each do |itemorder| %>
   <section id="order-<%=itemorder.order_id%>">
     <p>Order #<%= link_to("#{itemorder.order_id}", merchant_orders_path(itemorder.order_id)) %></p>
     <p>Date Created: <%= itemorder.order.created_at %></p>
+    <p>Status: <%= itemorder.order.status %></p>
```
Comment (lrs8810) [inline app/views/merchant/dashboard/index.html.erb:16]: Great addition! This makes the order status really clear to the merchant user.

**Comment by freeheeling** [comment]
Approved. Thank you.

**Pull request merged**